CMakeLists.txt

```diff
@@ -209,9 +209,25 @@ if (PLATFORM_LINUX OR PLATFORM_ANDROID)
 endif()
 
 if (NOT SKIP_TESTS)
-  set(SKIP_TESTS FALSE)
   enable_testing()
   set(CTEST_CONFIGURATION_TYPE ${CMAKE_BUILD_TYPE})
```

Comment:
These lines can be removed.
When `include(CTest)` is called, it will automatically call `enable_testing()`.
`set(CTEST_CONFIGURATION_TYPE ${CMAKE_BUILD_TYPE})` is not needed. I forgot to remove it.